manifest.json

```diff
@@ -142,6 +142,45 @@
                 "UpdateUser"
             ]
         },
+        "Kinesis": {
+            "source": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/kinesis/2013-12-02/api-2.json",
+            "documentation": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/kinesis/2013-12-02/docs-2.json",
+            "pagination": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/kinesis/2013-12-02/paginators-1.json",
+            "waiter": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/kinesis/2013-12-02/waiters-2.json",
+            "example": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/kinesis/2013-12-02/examples-1.json",
+            "api-reference": "https://docs.aws.amazon.com/kinesis/latest/APIReference",
+            "methods": [
+                "AddTagsToStream",
+                "CreateStream",
+                "DecreaseStreamRetentionPeriod",
+                "DeleteStream",
+                "DeregisterStreamConsumer",
+                "DescribeLimits",
+                "DescribeStream",
+                "DescribeStreamConsumer",
+                "DescribeStreamSummary",
+                "DisableEnhancedMonitoring",
+                "EnableEnhancedMonitoring",
+                "GetRecords",
+                "GetShardIterator",
+                "IncreaseStreamRetentionPeriod",
+                "ListShards",
+                "ListStreamConsumers",
+                "ListStreams",
+                "ListTagsForStream",
+                "MergeShards",
+                "PutRecord",
+                "PutRecords",
+                "RegisterStreamConsumer",
+                "RemoveTagsFromStream",
+                "SplitShard",
+                "StartStreamEncryption",
+                "StopStreamEncryption",
+                "StreamExists",
+                "StreamNotExists",
+                "UpdateShardCount"
```

Comment:
Do you really need all of these?

Reply:
No. I'm not yet sure what I need.

Reply:
I doubt I need any of the methods like `IncreaseStreamRetentionPeriod` since that would be managed by cloudformation.